lib/clickhouse-activerecord/schema_dumper.rb

```diff
@@ -147,8 +147,11 @@ def format_colspec(colspec)
     end
 
     def schema_limit(column)
-      return nil if column.type == :float
-      super
+      if column.type == :float
+        return 8 if column.sql_type == "Float64"
```

Comment:
Looks like CH supports `Float32` as well.

```suggestion
        return 4 if column.sql_type == "Float32"
        return 8 if column.sql_type == "Float64"
```

Reply:
Yeah the implied default for `float` is `Float32` currently:

https://github.com/huntresslabs/clickhouse-activerecord/blob/337231e4ba443f8dedbea4e8383e23dd06d601ae/lib/active_record/connection_adapters/clickhouse_adapter.rb#L89